sktime/utils/clustering_utils.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
```

Comment:
If they're only used for clustering I'd consider moving them into `sktime/clustering/base`